### PR TITLE
DF-851 Update buttons for search filters

### PR DESCRIPTION
**Pull request by JohnHeeryTNA**
Ticket URL: https://national-archives.atlassian.net/jira/software/projects/DF/issues/DF-851

## About these changes

Reducing the number of 'Update' buttons on Search Filters to reflect the layout on the OHOS project
(https://ohos-zgoyvpy-rasrzs7pi6sd4.uk-1.platformsh.site/search/catalogue/)

## How to check these changes

- Compare the layout of the search filters against the OHOS layout
- Test that the Update and Search within elements are functioning correctly.

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [x] Ensured that PR includes only commits relevant to the ticket
- [x] Waited for all CI jobs to pass before requesting a review
- [x] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

**Review by sdwilkes — APPROVED** [review]
Code looks good, approving this for testing on Develop. Thanks John.

File: templates/search/blocks/search_filters.html
Comment (TNA-Allan) [inline templates/search/blocks/search_filters.html:1]: templates/search/blocks/search_filters.html

Some Record Opening date bits are missing:

1. Field set is missing - over here previously https://github.com/nationalarchives/ds-wagtail/blob/b798f4c4338fd6ef11b6ac6a52660262438d03b0/templates/search/blocks/search_filters.html#L111C26-L111C26
2. Label is missing - over here previously https://github.com/nationalarchives/ds-wagtail/blob/b798f4c4338fd6ef11b6ac6a52660262438d03b0/templates/search/blocks/search_filters.html#L112C1-L114C34
3. To date is missing. over here previously
https://github.com/nationalarchives/ds-wagtail/blob/b798f4c4338fd6ef11b6ac6a52660262438d03b0/templates/search/blocks/search_filters.html#L133C1-L146C31

Comment (TNA-Allan) [inline templates/search/blocks/search_filters.html:1]: Some Bucket filters for Non Tna, Creator and Archive are missing-

Over here previously 
https://github.com/nationalarchives/ds-wagtail/blob/b798f4c4338fd6ef11b6ac6a52660262438d03b0/templates/search/blocks/search_filters.html#L154C1-L166C24

Comment (TNA-Allan) [inline templates/search/blocks/search_filters.html:1]: Our website results filters - time period, topic, format are missing.

Over here previously https://github.com/nationalarchives/ds-wagtail/blob/b798f4c4338fd6ef11b6ac6a52660262438d03b0/templates/search/blocks/search_filters.html#L95C1-L99C79

Comment (TNA-Allan) [inline templates/search/blocks/search_filters.html:1]: Search within is showing for - _Our website results_ tab, the condition is missing. Over here previously https://github.com/nationalarchives/ds-wagtail/blob/b798f4c4338fd6ef11b6ac6a52660262438d03b0/templates/search/blocks/search_filters.html#L34

Comment (TNA-Allan) [inline templates/search/blocks/search_filters.html:1]: Render hidden input missing - for form attributes outside that are outside the scope of this form element in this file. Previously over here 
https://github.com/nationalarchives/ds-wagtail/blob/b798f4c4338fd6ef11b6ac6a52660262438d03b0/templates/search/blocks/search_filters.html#L169C1-L170C80

Comment (TNA-Allan) [inline templates/search/blocks/search_filters.html:1]: Level filter divs, html showing in page-source for nonTna bucket. It shouldn't. 
view-source:http://127.0.0.1:8000/search/catalogue/?group=nonTna

```
<div class="govuk-accordion__section">
    <div class="govuk-accordion__section-header">
        <h2 class="govuk-accordion__section-heading tna-heading search-filters__heading">
            <span class="govuk-accordion__section-button" id="accordion-default-heading-4">
              Level
            </span>
        </h2>
    </div>
    <div id="accordion-default-content-4" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-4">
        
    </div>
</div>
```

The nonTna condition https://github.com/nationalarchives/ds-wagtail/blob/prototype/DF-851-update-buttons-combined/templates/search/blocks/search_filters.html#L145 is not wrapped for the whole Level div

File: templates/search/blocks/search_filters.html [hunk 1]
```diff
@@ -106,7 +112,7 @@ <h4 class="tna-heading search-filters__accordion-section-heading" id="record_cov
 
                                 </div>
 
-                                <input type="submit" value="Update" class="search-filters__submit">
+<!--                                <input type="submit" value="Update" class="search-filters__submit">-->
```
Comment (TNA-Allan) [inline templates/search/blocks/search_filters.html:115]: This comment is  starting from 1st col, it could be aligned under closing div to facilitate collapsing of html attributes in the editor.

File: templates/search/includes/filter.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
                     <a href="{% url 'search-website-long-filter-chooser' field_name=field.name %}?{{ request.GET.urlencode }}"  aria-label='See more'>See more</a>
                 {% endif %}
             {% endif %}
-            <input type="submit" value="Update" class="search-filters__submit">
+<!--            <input type="submit" value="Update" class="search-filters__submit">-->
```
Comment (TNA-Allan) [inline templates/search/includes/filter.html:16]: This comment is starting from 1st col, it could be aligned under closing div to facilitate collapsing of html attributes in the editor.

**Review by TNA-Allan — DISMISSED** [review]
Hi @JohnHeeryTNA , I think this is pretty much it. As discussed, let me know if I could create a branch off from this one with the changes where you can take a look and then merge it in your branch.

File: templates/search/blocks/search_filters.html
Comment (JohnHeeryTNA) [reply templates/search/blocks/search_filters.html:1]: These elements have now been addressed.

Comment (JohnHeeryTNA) [reply templates/search/blocks/search_filters.html:1]: These have been added.

Comment (JohnHeeryTNA) [reply templates/search/blocks/search_filters.html:1]: These have been added.

Comment (JohnHeeryTNA) [reply templates/search/blocks/search_filters.html:1]: condition added.

File: templates/search/blocks/search_filters.html [hunk 3]
```diff
@@ -67,51 +78,63 @@ <h2 class="govuk-accordion__section-heading tna-heading search-filters__heading"
                     <div class="search-filters__form-block">
                         <h3 class="tna-heading sr-only">Edit filters</h3>
 
-                        {% if form.group.value == 'tna' or form.group.value == 'digitised' or form.group.value == 'nonTna' %}
+                        {% if form.group.value == 'tna' or form.group.value == 'digitised' %}
```
Comment (TNA-Allan) [inline templates/search/blocks/search_filters.html:81]:
```suggestion
                        {% if form.group.value == 'tna' or form.group.value == 'digitised' or form.group.value == 'nonTna' %}
```
This condition of for Covering dates/Dates. No change required. Restore it.

File: templates/search/blocks/search_filters.html [hunk 4]
```diff
@@ -67,51 +78,63 @@ <h2 class="govuk-accordion__section-heading tna-heading search-filters__heading"
...
                         <div class="search-filters__accordion-section">
-                            <fieldset role="group" aria-describedby="from-date">
+                            <fieldset role="group" aria-describedby="opening-start-date">
```
Comment (TNA-Allan) [inline templates/search/blocks/search_filters.html:83]:
```suggestion
                            <fieldset role="group" aria-describedby="from-date">
```
Same for Covering dates/Dates

File: templates/search/blocks/search_filters.html [hunk 5]
```diff
@@ -67,51 +78,63 @@ <h2 class="govuk-accordion__section-heading tna-heading search-filters__heading"
...
                                 <legend>
-                                    <h4 class="tna-heading search-filters__accordion-section-heading" id="record_covering_date">Dates</h4>
+                                    <h4 class="tna-heading-s search-filters__accordion-section-heading" id="record_opening_date">Record opening date</h4>
```
Comment (TNA-Allan) [inline templates/search/blocks/search_filters.html:85]:
```suggestion
                                    <h4 class="tna-heading search-filters__accordion-section-heading" id="record_covering_date">Dates</h4>
```
Same for Covering dates/Dates. Restore previous values.

File: templates/search/blocks/search_filters.html [hunk 6]
```diff
@@ -67,51 +78,63 @@ <h2 class="govuk-accordion__section-heading tna-heading search-filters__heading"
...
                                 </legend>
 
-                                <span class="example-text" id="from-date">For example, 27 3 2007 or 2007</span>
-                                <div>{{ form.covering_date_from.label_tag }}</div>
-
-                                <div>
+                                <div class="search-filters__opening-date-section">
```
Comment (TNA-Allan) [inline templates/search/blocks/search_filters.html:88]: Same for Covering dates/Dates. Restore previous values.

File: templates/search/blocks/search_filters.html (hunk 3 above)
Comment (JohnHeeryTNA) [reply templates/search/blocks/search_filters.html]: restoring values.

File: templates/search/blocks/search_filters.html
Comment (JohnHeeryTNA) [reply templates/search/blocks/search_filters.html:1]: This has been addressed by **fix/DF-851-updates**

**Review by TNA-Allan — APPROVED** [review]
Looks good. Thanks @JohnHeeryTNA
